src/main/java/graphql/schema/idl/RuntimeWiring.java

```diff
@@ -0,0 +1,252 @@
+package graphql.schema.idl;
+
+import graphql.Assert;
+import graphql.schema.BuilderFunction;
+import graphql.schema.DataFetcher;
+import graphql.schema.GraphQLScalarType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.TypeResolver;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * A runtime wiring is a specification of data fetchers, type resolves and custom scalars that are needed
+ * to wire together a functional {@link GraphQLSchema}
+ */
+public class RuntimeWiring {
```

Comment:
I thought about calling it "Resolvers" but decided against it because DataFetchers are not known as Resolvers and hence it doesnt ring true for graphql-java